onnxruntime/wasm/api.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <emscripten.h>
+
+#include <stddef.h>
+
+namespace Ort {
+struct Session;
+}
+typedef Ort::Session* ort_session_handle_t;
```

Comment:
Do you want ort_session_handle_t to be opaque for this API? If yes, you might want to hide it's implementation detail. Same for ort_tensor_handle_t.

Reply:
there is no need to use opaque: users don't consume this API in C/C++. it will introduce reinterpret cast which can be avoid if do so.

Reply:
```suggestion
using ort_session_handle_t = Ort::Session*` ;
```

Reply:
change applied